guides/common/modules/proc_modifying-the-repository-sets.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Modifying_the_Repository_Sets_Status_in_Project{context}"]
+= Modifying the Repository Sets Status in {Project}
+
+Repository sets show available repositories to each host.
+If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts*, and select the host whose repository sets you want to modify.
+. Select a host and navigate to the *Repository sets* tab.
+On the *Repository sets* tab, there is a set of available repositories available to each host with a status of *Enabled* or *Disabled*.
+. Override the default status by selecting *Override to disabled*, *Override to enabled*, or *Reset to default* from the action menus on each table row.
+. Navigate to the *Overview* tab to *Content view details* to view the environment for the host.
++
+[NOTE]
+====
+For hosts not in a default view or a lifecycle environment, the repository tab looks different.
+It defaults to showing only the repositories that are relevant to the host.
+====
+. Select *Show all* or *Limit to environment* to view available repositories.
```

Comment:
I am unsure _why_ this is part of the procedure: users navigate somewhere, but there's no actions.

Maybe make it optional and reword to "verify..." or similar.